core/src/main/java/com/linecorp/armeria/common/ResponseEntity.java

```diff
@@ -62,6 +53,38 @@ static <T> ResponseEntity<T> of(ResponseHeaders headers, @Nullable T content, Ht
         return new DefaultResponseEntity<>(headers, content, trailers);
     }
 
+    /**
+     * Returns a newly created {@link ResponseEntity} with the specified {@link HttpStatus}.
+     */
+    static <T> ResponseEntity<T> of(HttpStatus status) {
+        return of(ResponseHeaders.of(status));
+    }
+
+    /**
+     * Returns a newly created {@link ResponseEntity} with the specified {@link HttpStatus} and
+     * {@code content}.
+     */
+    static <T> ResponseEntity<T> of(HttpStatus status, T content) {
+        return of(ResponseHeaders.of(status), content);
```

Comment:
This `of()` method has null check for `content`. But should I still add `requireNonNull()` here?

Reply:
> But should I still add requireNonNull() here?

No, you don't have to. 😉